executor/distsql.go

```diff
@@ -127,7 +141,17 @@ func zone(sctx sessionctx.Context) (string, int64) {
 	var name string
 	name = loc.String()
 	if name == "Local" {
-		name = "System"
+		path, err := filepath.EvalSymlinks("/etc/localtime")
```

Comment:
How is this function evaluated? If it is evaluated frequently, we need to cache the zone.

Reply:
Agree.